### ISSUE TITLE
Holograms over tables disapear

### ISSUE THREAD

**Issue by CloudeLecaw** [Matsuneitor/Roulette#10]
Today we saw that Roulette 6.3 removed the holograms over the tables

We tried to reload cmi
we tried to reload roulette
and we tried to reload citicenz

nothing helped
only solution here seems to be a server restart :\

After that we can aspect it will work a few hour aggain before the issue remain.

![Snag_1bd462c](https://user-images.githubusercontent.com/12080017/117076123-342aee00-ad36-11eb-83da-c24a6bb1545c.png)

**Comment by aematsubara** [comment]
So, they appear when doing /roulette create, but then dissapear?

**Comment by CloudeLecaw** [comment]
No we are not 100% what was deleting them.
We noticed thia just two times now in the 6.x Version.
For hologrammms we are using the new cmi apipart.

Well it was passend may 24 hours after we created the tables. We just went to the casino and wanted to play a last round before going to bed... when we reached the tables we noticed we where not able to start because the hololines over the tables where missing. We deleted the old tables after trying to reload citicenz, cmi, and also roulette without success of having new hololines and created new tables.   Since that it was just happening once.  We have no clue what removed the holograms. Also i just can tell you it normaly seems not to happen. We run the tables for some days now each time without any issue until we need to delete and rebuild them, because of the other issue with the armorstand that i allready reported.
